src/main/resources/com/google/api/codegen/java/discovery_fragment.snip

```diff
@@ -66,11 +66,18 @@
         JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
 
         {@ApiName} {@serviceName()} = new {@ApiName}.Builder(httpTransport, jsonFactory, credential)
-          .setApplicationName("Google Cloud Platform Sample")
-          .build();
+            .setApplicationName("Google Cloud Platform Sample")
+            .build();
 
         {@initialize(method)}
         {@methodCall(method)}
+        @if context.hasMediaDownload(method)
+
+          // {@TODO()} Download desired media content. For more information, see:
```

Comment:
My general approach has been that the sample comments should clarify the mechanics of the client library, but need not substitute for the API documentation. For instance, https://cloud.google.com/storage/docs/json_api/v1/objects/get#response says:

"By default, this responds with an object resource in the response body. If you provide the URL parameter alt=media, then it will respond with the object data in the response body."

From this I expect the user to understand that there's a switch between getting just metadata vs. getting the object data, and the purpose of the sample is just to indicate where and how that switch gets accomplished in client-library calling code. (Granted, the API docs could be clearer on this point.)